CHANGES.md

```diff
@@ -146,6 +146,7 @@
 
 ## I/Os
 
+* New experimental Firestore connector in Java SDK, providing sources and sinks to Google Cloud Firestore ([BEAM-8376](https://issues.apache.org/jira/browse/BEAM-8376)).
```

Comment:
BEAM-8376 is still not resolved. Does it mean that work is still in progress on this?

Reply:
The only remaining item for the Firestore connector is getting another gcp project configured and setup for firestore integration tests. I've been in contact with someone from the Firestore backend team to sponsor and get a project setup.

The integration tests are authored and do succeed if they are able to connect to a Firestore in Native Mode database, however the current database in the `apache-beam-testing` is Firestore in Datastore Mode.

Reply:
BEAM-8376 has been marked resolved.